vortex-app/vortex-app-api/src/main/resources/application.yaml

```
@@ -158,6 +175,11 @@ app:
             - ORG_ADMIN
             - ORG_MEMBER
   gateway:
+    consoleconnect:
+      base-url: ${GATEWAY_DOWN_STREAM_BASE_URL:https://api.test.consoleconnect.com}
```

```suggestion
      base-url: ${GATEWAY_DOWN_STREAM_BASE_URL:https://api.consoleconnect.com}
```

Given this is an open source project, I would prefer to have public urls only.

---

we can config base-url in different env ,